fhir-bucket/src/main/java/com/ibm/fhir/bucket/scanner/CosScanner.java

```diff
@@ -149,7 +153,7 @@ public void mainLoop() {
                 }
             } catch (Exception x) {
                 // Just catch and log so we don't break the main loop
-                logger.severe("Error during COS scan: " + x.getMessage());
+                logger.log(Level.SEVERE, "Error during COS scan", x);
```

Comment:
I think I was finding the stack traces annoying :-)

Reply:
hopefully we've fixed the cases where we see this over and over and want to ignore it... without the stack trace its really hard to know what went wrong